feathr_project/setup.py

```diff
@@ -3,12 +3,10 @@
 
 # Use the README.md in root project
 root_path = Path(__file__).resolve().parent.parent
-long_description = (root_path / "README.md").read_text()
 
 setup(
     name='feathr',
     version='0.3.2',
-    long_description=long_description,
     long_description_content_type="text/markdown",
```

Comment:
dont' change this. This wont' work for pypi.

Reply:
Reverted